VAMobile/documentation/docs/Engineering/BackEnd/Features/MaintenanceWindows.md

```diff
@@ -6,7 +6,7 @@ Maintenance windows are periods of time during which specific vets-api functiona
 
 ## Back-end
 
-Unlike most of the data used in the vets-api engine, `MaintenanceWindows` come from the vets-api database and not from upstream servers. They're added there by developers on other teams. They contain the external service name, start time, and end time. The mobile app requests maintenance windows via the maintenance windows controller, which searches the database for `MaintenanceWindow` records that end in the future and creates a `ServiceGraph` object to process those windows into a format that is more easily used by the front-end.
+Unlike most of the data used in the vets-api engine, `MaintenanceWindows` come from the vets-api database and not from upstream servers. They contain the external service name, start time, and end time. They're added there by developers on other teams to PagerDuty, and a Sidekiq job populates the database based on PagerDuty. There is a [white list](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/blob/9864e007626ceb2cd5c92b8c8838af1dbb3b4b7d/apps/vets-api/prod/values.yaml#L2169) of maintenance windows on GitHub that the sidekiq job uses to connect to PagerDuty and populate the maintenance windows. The mobile app requests maintenance windows via the maintenance windows controller, which searches the database for `MaintenanceWindow` records that end in the future and creates a `ServiceGraph` object to process those windows into a format that is more easily used by the front-end.
```

Comment:
This is a good addition. I think the sentence starting "They're added there by developers on other teams to PagerDuty" might be more clearly worded as "Maintenance windows are added to PagerDuty by other teams, and a vets-api sidekiq job polls PagerDuty and populates MaintenanceWindow database records with the response."

Reply:
Ah ya that wording is much clearer